src/api/version.ts

```diff
@@ -163,7 +176,151 @@ export namespace VersionHelper {
     }
   }
 
-  function removeVPrefix(version: string): string {
-    return version.startsWith('v') ? version.substring(1) : version
+  /**
+   * Returns latest chectl version for the given channel.
+   */
+  export async function getLatestChectlVersion(channel: string): Promise<string | undefined> {
+    if (getProjectName() !== 'chectl') {
+      return
+    }
+
+    const axiosInstance = axios.create({
+      httpsAgent: new https.Agent({})
```

Comment:
if user has proxy settings, how does it work ?

Reply:
AFAIK `axios` reads corresponding env variables.